osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -167,6 +172,11 @@ private double computeSpeedValue(ScoreInfo score, OsuDifficultyAttributes attrib
                 // We want to give more reward for lower AR when it comes to aim and HD. This nerfs high AR and buffs lower AR.
                 speedValue *= 1.0 + 0.04 * (12.0 - attributes.ApproachRate);
             }
+            else if (score.Mods.Any(h => h is OsuModTraceable))
+            {
+                // More reward for speed because speed on Traceable is annoying
```

Comment:
Was there any kind of survey if that's even true?

Reply:
it can be changed easily anyways
it needs some bonus to make it worth playing